### PR TITLE
feat(aws): scrape cloudformation stacks

### PR DESCRIPTION
**Pull request by adityathebe**
resolves: https://github.com/flanksource/config-db/issues/506

depends on: https://github.com/flanksource/is-healthy/pull/62

### CODE REVIEW

**Comment by adityathebe** [comment]
image

File: scrapers/aws/aws.go [hunk 1]
```diff
@@ -183,6 +184,42 @@ func (aws Scraper) sqs(ctx *AWSContext, config v1.AWS, results *v1.ScrapeResults
...
+	ctx.Logger.V(2).Infof("scraping CloudFormation stacks")
+
+	client := cloudformation.NewFromConfig(*ctx.Session)
+	stacks, err := client.ListStacks(ctx, &cloudformation.ListStacksInput{})
+	if err != nil {
+		results.Errorf(err, "failed to list CloudFormation stacks")
+		return
+	}
+
+	for _, stack := range stacks.StackSummaries {
```
Comment (moshloop) [inline scrapers/aws/aws.go:201]: Can we also link to the resources created by the stack ?

Comment (adityathebe) [reply scrapers/aws/aws.go:201]: I think we can do that in the registry?

```yaml
 - filter: config_type == 'AWS::CloudFormation::Stack'
   expr: |
     [{"labels": {"aws:cloudformation:stack-id": config["StackId"]}}].toJSON()
```

Comment (moshloop) [reply scrapers/aws/aws.go:201]: I actually think the parent of the AWS resource should be the Cloudformation stack - the same way we do helm / argo

Comment (adityathebe) [reply scrapers/aws/aws.go:201]: yes this will set the cloudformation stack has the parent.

Comment (adityathebe) [reply scrapers/aws/aws.go:201]: Tested


image

**Pull request merged**